src/components/AddPlaidBankAccount.js

```diff
@@ -123,9 +125,9 @@ class AddPlaidBankAccount extends React.Component {
                             <ActivityIndicator color={themeColors.spinner} size="large" />
                         </View>
                     )}
-                    {Boolean(this.props.plaidData.error) && (
+                    {(Boolean(this.props.plaidData.error) || Boolean(this.props.plaidData.errors)) && (
```

Comment:
```suggestion
                    {Boolean(plaidDataErrorMessage) && (
```

I think that's pretty clean, and using the boolean constructor ensures there won't be a problem if plaidDataErrorMessage is an empty string, i.e https://github.com/facebook/react-native/issues/29980